ClientLibrary/Quickstarts/netcore/DualGraphGroupMembershipSample/Program.cs

```diff
@@ -0,0 +1,263 @@
+﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Microsoft.VisualStudio.Services.Client;
+using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.VisualStudio.Services.Graph.Client;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.Services.Common;
+using Newtonsoft.Json;
+
+namespace DualGraphGroupMembershipSample
+{
+    class Program
+    {
+        //============= Config [Edit these with your settings] =====================
+        internal const string vstsCollectionUrl = "https://peaky.fstest10.tfsallin.net"; //"https://myaccount.visualstudio.com";  //change to the URL of your VSTS account; NOTE: This must use HTTPS
+        internal const string clientId = "872cd9fa-d31f-45e0-9eab-6e460a02d1f1";//"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx";         //change to your app registration's Application ID
+        internal const string replyUri = "urn:ietf:wg:oauth:2.0:oob";                           //change to your app registration's reply URI
+        internal const string groupDisplayName = "test1";//"vsts group display name";              //change to the group you want to get all users principal names
+        //==========================================================================
+
+        internal const string VstsResourceId = "499b84ac-1321-427f-aa17-267ca6975798"; //Constant value to target VSTS. Do not change
+        internal const string graphResourceId = "https://graph.microsoft.com";         //Constant value to target Microsoft Graph API. Do not change
+
+        public static void Main(string[] args)
+        {
+            AuthenticationContext ctx = GetAuthenticationContext(null);
+            AuthenticationResult result = null;
+            try
+            {
+                //PromptBehavior.RefreshSession will enforce an authn prompt every time. NOTE: Auto will take your windows login state if possible
+                result = ctx.AcquireTokenAsync(VstsResourceId, clientId, new Uri(replyUri), new PlatformParameters(PromptBehavior.Always)).Result;
+
+                
+
+                //connect to graph API with ADAL access token
+                VssConnection connection = new VssConnection(new Uri(vstsCollectionUrl), new VssOAuthCredential(result.AccessToken));
```

Comment:
This API is marked obsolete, change to Microsoft.VisualStudio.Services.OAuth.VssOAuthAccessTokenCredential